cogs/member_stats.py

```diff
@@ -30,5 +156,25 @@ async def m_stats(self, ctx):
                     if member.status != discord.Status.offline:
                         member_online_count += 1
 
-            await ctx.send(f'Total members: {true_member_count}')
-            await ctx.send(f'Online members: {member_online_count}')
+            from_zone = tz.gettz('UTC')
+            to_zone = tz.gettz('America/Los_Angeles')
+
+            utc_now = datetime.utcnow()
+            my_now = utc_now.replace(tzinfo=from_zone)
+            now = my_now.astimezone(to_zone)
+
+            tsMonth = now.strftime("%m")
+            tsDay = now.strftime("%d")
+            tsYear = now.strftime("%Y")
```

Comment:
This is just a nitpick, but maybe this could instead be ordered so that it's consistent with the formatted string below:
```py
tsYear = now.strftime("%Y")
tsMonth = now.strftime("%m")
tsDay = now.strftime("%d")
```